src/render.js

```diff
@@ -138,15 +171,17 @@ const initSettings = () => {
 
   // Populate the dialect selector
   const dialectSelector = document.getElementById('dialectSelection');
-  Generator.SUPPORTED_DIALECTS.forEach((language) => {
-    const opt = document.createElement('option');
-    opt.value = language;
-    opt.innerHTML = language;
-    dialectSelector.appendChild(opt);
-    if (gherkinDialect === language) {
-      opt.selected = 'selected';
-    }
-  });
+  if (Generator.SUPPORTED_DIALECTS) {
+    Generator.SUPPORTED_DIALECTS.forEach((language) => {
+      const opt = document.createElement('option');
+      opt.value = language;
+      opt.innerHTML = language;
+      dialectSelector.appendChild(opt);
+      if (gherkinDialect === language) {
+        opt.selected = 'selected';
+      }
+    });
+  }
```

Comment:
could you drop this back off? the only reason this would occur is if someone's current dependencies are out of date, in which case we'd want this error so they can't try to build with a bad dependency version

Reply:
done